src/main/java/org/opensearch/knn/quantization/models/quantizationState/OneBitScalarQuantizationState.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.quantization.models.quantizationState;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.opensearch.knn.quantization.models.quantizationParams.ScalarQuantizationParams;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+
+/**
+ * OneBitScalarQuantizationState represents the state of one-bit scalar quantization,
+ * including the mean values used for quantization.
+ */
+@Getter
+@NoArgsConstructor // No-argument constructor for deserialization
+@AllArgsConstructor // Constructor with all arguments
+public final class OneBitScalarQuantizationState implements QuantizationState {
+    private ScalarQuantizationParams quantizationParams;
+    private float[] meanThresholds;
+    private static final long serialVersionUID = 1L; // Version ID for serialization
+    private static final int CURRENT_VERSION = 1; // Current version of SQParams
+
```

Comment:
Can we add a similar type of description from MultiBitScalarQuantization here as it helped me a lot to understand about the class?